internal/command/commands/serviceset.go

```diff
@@ -30,6 +30,7 @@ func (s *ServiceSet) Run(args []string) int {
 	envName := flagSet.String("env", "", "name of environment to deploy the service-set in")
 	platform := flagSet.String("platform", "", "platform of environment to deploy the service-set in")
 	force := flagSet.Bool("force", false, "forcefully deploy service-set into the Env")
+	configStoreNamespace := flagSet.String("d11-config-store-namespace", "", "config store branch/tag to use")
```

Comment:
why we are prefixing `d11` in flag?

Reply:
this is to be consistent w.r.t service deploy command/API. once we finalise on what type of entity is config store, we can give it a appropriate name. for now we are making it a dream11 specific entity.
cc: @akshaypatidar1999